src/Makefile.am

```diff
@@ -16,7 +16,9 @@ LOADABLE_MODULES_SOURCES = \
 	LoadableModules.cc \
 	LoadableModules.h
 
-SUBDIRS = mem time debug base anyp helper dns html ftp parser comm error eui acl format clients sbuf servers fs repl store DiskIO proxyp
+# order is important. If anything in a subdir requires a convenience library
+# from another subdir, then it must appear later in this list
```

Comment:
If you insist on including this hint, let's polish it to clarify what "requires a library" means here (and generalize):

```suggestion
# subdir Y that requires a file built in subdir X must appear after X
```